tests/test_resources/test_clusters/test_cluster.py

```diff
@@ -896,3 +897,42 @@ def test_switch_default_env(self, cluster):
         # set it back
         cluster.default_env = test_env
         cluster.delete(new_env.name)
+
+    @pytest.mark.level("local")
+    @pytest.mark.clustertest
+    @pytest.mark.skip(
+        "Stopping and restarting the server mid-test causes some errors, need to fix"
+    )
+    def test_enable_and_disable_observability(self, cluster):
+        import json
+
+        # check that by default cluster telemetry is enabled
+        cluster_config_on_cluster = cluster.run(
```

Comment:
why not use `cluster.config_for_rns`?

Reply:
If we won't save the `enable_observability`on the local cluster_config on the cluster, this test will be modified. (Probably will use cluster.status() to get the value of `enable_observability`)